src/services/MilestoneService.js

```diff
@@ -387,6 +387,19 @@ class MilestoneService {
     let etherScanUrl;
 
     try {
+      const response = await milestones.find({
+        query: {
+          campaignId: milestone.campaignId,
+          title: milestone.title,
```

Comment:
@mdehghani Is it possible to implement this backend logic here too?
https://github.com/Giveth/feathers-giveth/blob/c3926652bde6aee57c163fe2630b23502789b2dd/src/services/milestones/checkMilestoneName.js#L23